Training/python/scaling_utils.py

```diff
@@ -173,11 +173,11 @@ def compute_mean(sums, counts, aggregate=True, *file_range):
     if aggregate:
         if file_range:
             assert len(file_range) == 2 and file_range[0] <= file_range[1]
-            return sums[file_range[0]:file_range[1]].sum()/counts[file_range[0]:file_range[1]].sum()
+            return sums[file_range[0]:file_range[1]].sum()/counts[file_range[0]:file_range[1]].sum() if counts[file_range[0]:file_range[1]].sum()>1 else 0.
         else:
-            return sums.sum()/counts.sum()
+            return sums.sum()/counts.sum() if counts.sum()>1 else 0.
```

Comment:
I am thinking that having 0(1) for mean(std) as return values in case of few counts might be misleading for further plotting and mean/std analysis. Would it be more suitable to return `None` and have a printout like "low statistics in {var} for mean/std computation" in that case? 

Of course having `mean=0` and `std=1` values stored in scaling json is OK in the sense that effectively it won't apply scaling to it. But in `DataLoader` there is also [clamping](https://github.com/cms-tau-pog/TauMLTools/blob/d233f566dfd733fa01cdf7986feb816882555b09/Training/interface/DataLoader_main.h#L320), therefore in that case feature by default will be clamped to interval [-5, 5], which might not be entirely correct. So having it as a `None` would be a sign that some manual treatment is needed.